cmake/macros/macro_deal_ii_add_test.cmake

```diff
@@ -365,9 +369,17 @@ FUNCTION(DEAL_II_ADD_TEST _category _test_name _comparison_file)
           ${TEST_LIBRARIES} ${TEST_LIBRARIES_${_build}}
           )
 
+        SET(_performance_test_macro_definition "")
+        IF(ENABLE_PERFORMANCE_TESTS)
+          SET(_performance_test_macro_definition "ENABLE_PERFORMANCE_TESTS")
```

Comment:
I'm not crazy about this. I think what you're doing is allowing `ENABLE_PERFORMANCE_TESTS` to be set to anything other than `TRUE` or `FALSE`, and then below you pass on this definition. Is that right? I think you should only allow `TRUE`/`FALSE`/`ON`/`OFF`.

Reply:
This statement will expand the string "${_performance_test_macro_definition}" to "ENABLE_PERFORMANCE_TESTS" if and only if the CMake variable ${ENABLE_PERFORMANCE_TESTS} is defined and equates to a (CMake) true value. This is then later used to have a preprocessor definition equivalent to
```
#define ENABLE_PERFORMANCE_TESTS
```
so that I can use `#ifdef [...]`

This behavior is similar to what we do in our `base/config.h.in` with a statement like `#cmakedefine ENABLE_PERFORMANCE_TESTS`.

Reply:
Also no longer part of this pull request :joy: